Snakefile

```diff
@@ -201,8 +201,8 @@ rule percentile_ranks:
             # remove only microarray derived multiple percentile ranks
             # (per array design <accession>_<arraydesign>-percentile-ranks.tsv)
             rm -f {wildcards.accession}_*-percentile-ranks.tsv
-        else
-            mv ${{percentile_ranks[0]}} {output.percentile_ranks_merged}
+            #else
+            #mv ${{percentile_ranks[0]}} {output.percentile_ranks_merged}
```

Comment:
Lets just remove this. Although strictly speaking, I would suggest a separate PR for this, then it is easier to revert if needed (and safer to remove the comments).

Reply:
But given the small change, lets just leave it here.

Reply:
Done